### ISSUE TITLE
blocking resources may cause high cpu usage on some site

### ISSUE THREAD

**Issue by AdClear247** [uBlockOrigin/uBlock-issues#1592]
### Prerequisites

- [ ] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [ ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [ ] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported
- [ ] The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser
    - If the issue is still present after wholly disabling uBO in the browser, then the issue is unrelated to uBO
- I tried to reproduce the issue when...
    - [ ] uBO is the only extension
    - [ ] uBO with default lists/settings
    - [ ] using a new, unmodified browser profile
- [ ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

Basically you should find some way to prevent high cpu usage by scripts of site when we block some elements like images. I think redirection would work perfectly.

### A specific URL where the issue occurs

https://forum.klix.ba/

### Steps to Reproduce

1. go to url
2. block all images using whatever method
3. refresh page

### Expected behavior:

should just block all images

### Actual behavior:

blocks all images but counter shows increasing count >1k and in network monitor we can observe same image trying to reload

### Your environment

* uBlock Origin version: 1.35.2
* Browser Name and version: Firefox 90.0a1
* Operating System and version: Windows 10 20H2

**Comment by uBlock-user** [comment]
Issue is very intermittent, can't reproduce consistently, seems the site keeps polling the blocked images when it happens.

**Comment by uBlock-user** [comment]
> I think redirection would work perfectly.

Add `*$image,redirect-rule=1x1.gif,domain=forum.klix.ba` to my filters
